papertty/drivers/driver_it8951.py

```diff
@@ -306,11 +307,49 @@ def display_area(self, x, y, w, h, display_mode):
         self.write_data_half_word(h)
         self.write_data_half_word(display_mode)
 
-    def draw(self, x, y, image, update_mode_override=None):
+    def draw_multi(self, imageArray):
+
+        """This function performs multiple draws in a single panel refresh"""
+
+        #First, calculate the bounds of the panel area which is being refreshed.
+        #ie. a rectangle within which all of the images in imageArray would fit.
```

Comment:
nit: I appreciate all the comments very much, but they would be a bit more readable if you added a space after each `#` :)